src/components/Pattern/Pattern.stories.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import Pattern from './Pattern';
+
+export default {
+  title: 'Components/Pattern',
+  component: Pattern
+};
+
```

Comment:
Add the different variants you want to view in here. For example:

`<Pattern color="midblue" size="M" />`

Will make it easier to preview and test the different variantions.